zipline/utils/events.py

```diff
@@ -603,6 +605,11 @@ class time_rules(object):
     every_minute = Always
 
 
+class calendars(object):
+    US_EQUITIES = sentinel('us_equities_calendar')
```

Comment:
Nope, only that I didn't know that was the convention 😄 . I do think "us_equities_calendar" is a little clearer, but am also fine with changing it to be the same as "US_EQUITIES" if that's more consistent.